components/script/dom/bindings/mozmap.rs

```diff
@@ -69,12 +122,12 @@ impl<T, C> FromJSValConvertible for MozMap<T>
                 return Err(());
             }
 
-            let property = match T::from_jsval(cx, property.handle(), config.clone())? {
+            let property = match V::from_jsval(cx, property.handle(), config.clone())? {
```

Comment:
I have a problem with using JS_GetOwnPropertyDescriptorById. To call that function I should have MuableHandle<PropertyDescriptor> and I am not sure how to obtain one in a way that will make the PropertyDescriptor rooted properly.  AFAIK standard way to call such functions is doing sth like `rooted!(in(cx) let mut desc = UndefinedValue())` and using desc.handle_mut() but it gives jsapi::MutableHandle<js::jsapi::Value> instead of MutableHandle<PropertyDescriptor>. I would appreciate some pointers